api/getambassador.io/v2/Host.proto

```diff
@@ -96,3 +113,24 @@ message ACMEProviderSpec {
   // This is normally set automatically
   string registration = 4;
 }
+
+message RequestPolicy {
+  // How shall we handle insecure requests?
```

Comment:
We should explain how we decide whether a given request is secure or not. I believe we had spoken about  separating the idea of a request being secure/insecure from the way it happens to arrive, i.e. classifying a request as insecure should be based on X-Forwarded-Proto, not port, so that if e.g. an ELB terminates TLS and sends it to ambassador over a cleartext connection, it is able to still be considered secure if that's what the user wants.

Reply:
In fact, it _is_ completely based on `X-Forwarded-Proto`! with the single exception that a  `requestPolicy.insecure.additionalPort` is defined to always be insecure.